core-bundle/src/Picker/AbstractTablePickerProvider.php

```diff
@@ -137,8 +137,9 @@ public function supportsContext($context): bool
         $this->framework->initialize();
         $this->framework->createInstance(DcaLoader::class, [$table])->load();
 
-        return $this->getDataContainer() === DataContainer::getDriverForTable($table)
-            && 0 !== \count($this->getModulesForTable($table));
+        return ($this->getDataContainer() === DataContainer::getDriverForTable($table)
+                || $this->getDataContainer() === $GLOBALS['TL_DCA'][$table]['config']['dataContainer']
+            ) && 0 !== \count($this->getModulesForTable($table));
```

Comment:
```suggestion
        $dc = $this->getDataContainer();

        return ($dc === DataContainer::getDriverForTable($table) || $dc === $GLOBALS['TL_DCA'][$table]['config']['dataContainer'])
            && 0 !== \count($this->getModulesForTable($table));
```

In case the `getDataContainer()` call is expensice, we should probably store its result in a variable.

Reply:
Changed in 9424537b68da0134f578619579cfd818f087550a